build.fsx

```diff
@@ -93,4 +93,38 @@ pipeline "Verify" {
     runIfOnlySpecified true
 }
 
+let referenceProjectDir = "./tests/FSharp.ApiDocs.Tests/files/ReferenceProject"
+
+// TODO: Revisit to see how we can use `dotnet watch` to run a local version of the tool
```

Comment:
This worked for me:

```bash
dotnet watch run --project .\src\fsdocs-tool\fsdocs-tool.fsproj -- watch --input "$HOME/projects/fantomas/docs" --eval --projects "$HOME/projects/fantomas/src/Fantomas.Core/Fantomas.Core.fsproj"
```

(you probably don't need `--eval`)

Reply:
Thanks for the hint I will have a look.